src/playturn.cpp

```diff
@@ -190,7 +190,7 @@ turn_info::PROCESS_DATA_RESULT turn_info::process_network_data(const config& cfg
 		}
 
 		auto disp_set_team = [](int side_index) {
-			const bool side_changed = display::get_singleton()->viewing_team() != side_index;
+			const bool side_changed = static_cast<int>(display::get_singleton()->viewing_team()) != side_index;
```

Comment:
Instead of using a cast, change the type of `side_index` to `std::size_t`.

Reply:
the cast seems fine to me.

Reply:
Of course casting is "fine", but converting a `size_t` to `int` and casting it back later is uglier than just keeping it as `size_t`.